unity-client/Assets/Scripts/MainScripts/DCL/Bridges/GIFProcessingBridge/GIFProcessingBridge.cs

```diff
@@ -37,10 +31,12 @@ void Awake()
             }
 
             i = this;
+
+            KernelConfig.i.EnsureConfigInitialized().Then(config => { jsGIFProcessingEnabled = config.gifSupported; });
```

Comment:
I think we should spread the knowledge of this promise class so we make more uses of this pattern in C#. 

Usually, we use callbacks for this behaviour. Having this pattern here and there breaks the project code consistency. I like it but I think we should aim to standardize it or something.